File: ZeroJudge/C++/a/a004/a004.cpp
#include <iostream>

using namespace std;

int main() {
	int year = 0;

	while (cin >> year) {
		if ((year % 4 == 0 && year % 100 != 0) || year % 400 == 0) {
			cout << "¶|¦~" << endl;
		}
		else {
			cout << "¥­¦~" << endl;
		}
	}

	return 0;
}

File: ZeroJudge/Java/src/a/a004/a004.java
package a.a004;

import java.util.Scanner;

public class a004 {
    public static void main(String args[]){
        Scanner input = new Scanner(System.in);

        while (input.hasNext()){
            int year = input.nextInt();

            if((year % 4 == 0 && year % 100 != 0) || year % 400 == 0){
                System.out.println("閏年");
            }
            else{
                System.out.println("平年");
            }

        }
    }
}
